### PR TITLE
Custom Fields Segmentation for TNE

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/managing-contacts/segmenting-your-contacts.md [hunk 1]
```diff
@@ -11,132 +11,64 @@ seo:
...
-Segments are dynamically updated over time as you add contacts who meet the criteria of your segment, or as the traits of your contacts change. For example, a segment with the criteria ”opened an email within 30 days” will evolve as contacts engage (or don’t engage) with your email.
 
-## 	Creating a Segment
-
-You can create a new segment with multiple conditions based on the information you have stored about your contacts, and based on some of the information SendGrid has stored about your Marketing Campaigns.
+You can use the information stored on your contacts from [custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) or reserved fields to create segments. Segments are similar to contact lists, except they update in real-time when a contact’s data matches the criteria that you set. When you segment your audience, you are able to create personalized Marketing Campaigns that directly address the wants and needs of your particular audience Segments can pull from *All Contacts* or any of your Contact lists. [Custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) allow you to use unique information to identify contacts for different segments.
```
Comment (xtineledlie) [inline content/docs/ui/managing-contacts/segmenting-your-contacts.md]:
```suggestion
You can use the information stored on your contacts from [custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) or reserved fields to create segments. Segments are similar to contact lists, except they update in real-time when a contact’s data matches the criteria that you set. When you segment your audience, you are able to create personalized Marketing Campaigns that directly address the wants and needs of your particular audience. Segments can pull from *All Contacts* or any of your Contact lists. [Custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) allow you to use unique information to identify contacts for different segments.
```

Comment (xtineledlie) [reply content/docs/ui/managing-contacts/segmenting-your-contacts.md]: (There was just a missing period that I added)

File: content/docs/ui/managing-contacts/segmenting-your-contacts.md [hunk 2]
```diff
@@ -11,132 +11,64 @@ seo:
...
+1. Select the **Create** drop-down.
+1. Click **New Segment**.
+1. Enter a Segment Name.
 1. Choose **Segment all contacts** or **Segment an existing list**.
-1. Enter a *Segment Name*.
-1. From the *Condition* drop-down, select **Engagement:Clicks**.
-1. From the *Criteria* drop-down. select **clicked**.
-1. From the last drop-down, select the name of the campaign you want to use for the segment.
+1. From the targeting drop-down, select **City**. 
+1. Select the operator **is**.
```
Comment (xtineledlie) [inline content/docs/ui/managing-contacts/segmenting-your-contacts.md]:
```suggestion
1. Select the operator **IS**. 
```

Comment (xtineledlie) [reply content/docs/ui/managing-contacts/segmenting-your-contacts.md]: Not 100% sure

File: content/docs/ui/managing-contacts/segmenting-your-contacts.md [hunk 3]
```diff
@@ -11,132 +11,64 @@ seo:
...
-1. From the *Criteria* drop-down. select **clicked**.
-1. From the last drop-down, select the name of the campaign you want to use for the segment.
+1. From the targeting drop-down, select **City**. 
+1. Select the operator **is**. 
+1. In the date field, type **Denver** and then click **Save Segment**.  
 
 <call-out>
 
-The *Condition* drop-down contains required CSV fields, custom fields, and internal event data to help you get your email campaigns to the correct audience. The *Criteria* drop-down contains operators that tell the segment *how* to handle the data specified by the condition.
+The *Field* drop-down contains Reserved Fields, or Custom Fields as designated by the CSVs you have uploaded. The *Operator* drop-down contains operators that tell the segment *how* to handle the data specified by the condition.
```
Comment (xtineledlie) [inline content/docs/ui/managing-contacts/segmenting-your-contacts.md]:
```suggestion
The *field* drop-down contains Reserved Fields, or Custom Fields as designated by the CSVs you have uploaded. The *operator* drop-down contains operators that tell the segment *how* to handle the data specified by the condition.
```

Comment (xtineledlie) [reply content/docs/ui/managing-contacts/segmenting-your-contacts.md]: (To match the steps)?

File: content/docs/ui/managing-contacts/segmenting-your-contacts.md [hunk 4]
```diff
@@ -11,132 +11,64 @@ seo:
...
-  </tr>
-</table>
-
-<br/>
-When using more than one condition to segment your contacts, you can select `AND` or `OR` as options, where `AND` requires both conditions to be true and `OR` requires either condition to be true. Once you’ve selected the Condition and the Criteria, you can layer additional conditions in the segment to narrow the results to a more targeted audience.
-
-<call-out>
-
-You can add up to 15 different conditions per segment.
+Depending on which field is selected, the operator you can choose from differ. The operator available to assign to fields will depend on the field type (Text, Date, or Number). For example, a "Number” field will show `IS`, `IS NOT`, `IS GREATER THAN`, `IS LESS THAN`, `IS EQUAL OR GREATER THAN`, and `IS EQUAL OR LESS THAN` as options whereas a “Text” field will show `CONTAINS`, `DOES NOT CONTAIN`, `IS` and `IS NOT` as options.
```
Comment (xtineledlie) [inline content/docs/ui/managing-contacts/segmenting-your-contacts.md]:
```suggestion
Depending on which field is selected, the *operator* you can choose from differ. The *operator* available to assign to fields will depend on the *field* type (Text, Date, or Number). For example, a "Number” field will show `IS`, `IS NOT`, `IS GREATER THAN`, `IS LESS THAN`, `IS EQUAL OR GREATER THAN`, and `IS EQUAL OR LESS THAN` as options whereas a “Text” field will show `CONTAINS`, `DOES NOT CONTAIN`, `IS` and `IS NOT` as options.
```

Comment (xtineledlie) [reply content/docs/ui/managing-contacts/segmenting-your-contacts.md]: (To match the steps, maybe?)

File: content/docs/ui/managing-contacts/segmenting-your-contacts.md [hunk 5]
```diff
@@ -11,132 +11,64 @@ seo:
...
-Segments are dynamically updated over time as you add contacts who meet the criteria of your segment, or as the traits of your contacts change. For example, a segment with the criteria ”opened an email within 30 days” will evolve as contacts engage (or don’t engage) with your email.
 
-## 	Creating a Segment
-
-You can create a new segment with multiple conditions based on the information you have stored about your contacts, and based on some of the information SendGrid has stored about your Marketing Campaigns.
+You can use the information stored on your contacts from [custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) or reserved fields to create segments. Segments are similar to contact lists, except they update in real-time when a contact’s data matches the criteria that you set. When you segment your audience, you are able to create personalized Marketing Campaigns that directly address the wants and needs of your particular audience. Segments can pull from *All Contacts* or any of your Contact lists. [Custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) allow you to use unique information to identify contacts for different segments.
```
Comment (xtineledlie) [inline content/docs/ui/managing-contacts/segmenting-your-contacts.md]:
```suggestion
You can use the information stored on your contacts from [custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) or reserved fields to create segments. Segments are similar to contact lists, except they update in real-time when a contact’s data matches the criteria that you set. When you segment your audience, you are able to create personalized Marketing Campaigns that directly address the wants and needs of your particular audience. Segments can pull from **All Contacts** or any of your Contact lists. [Custom fields]({{root_url}}/ui/managing-contacts/custom-fields/) allow you to use unique information to identify contacts for different segments.
```

File: content/docs/ui/managing-contacts/segmenting-your-contacts.md (hunk 2 above)
Comment (Whatthefoxsays) [reply content/docs/ui/managing-contacts/segmenting-your-contacts.md]: I'm matching the UI, so unfortunately, it's lowercase.

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**